### ISSUE TITLE
Millenial ad network for iOS

### ISSUE THREAD

**Issue by bburgess1966** [gideros/giderosplugins#3]
Add in the functionality for the Millenial ad network for iOS on the 'Ads Interface plugin'.   It is already in place for Android, but not yet for iOS.

**Comment by cyberience** [comment]
yes, is this possible? or any guidance to add one my self? I am not a C guy so its not so simple for me.
